Flaky integration test scheduler/test_generic_driver.py::test_kill[OpenPBSDriver]

**Describe the bug**
There is a test that fails sometimes, and seems to be flaky. 
```
=================================== FAILURES ===================================
___________________________ test_kill[OpenPBSDriver] ___________________________

driver = <ert.scheduler.openpbs_driver.OpenPBSDriver object at 0x7f369aff9100>
tmp_path = PosixPath('/private/f_scout_ci/pytest-tmp/tmp.VZAVfoiCxW/test_kill_OpenPBSDriver_0')

    async def test_kill(driver, tmp_path):
        os.chdir(tmp_path)
        aborted_called = False
    
        expected_returncodes = [1]
        if isinstance(driver, OpenPBSDriver):
            expected_returncodes = [128 + signal.SIGTERM, 256 + signal.SIGTERM]
    
        if isinstance(driver, LocalDriver):
            expected_returncodes = [-signal.SIGTERM]
    
        async def started(iens):
            nonlocal driver
            await driver.kill(iens)
    
        async def finished(iens, returncode, aborted):
            assert iens == 0
            assert returncode in expected_returncodes
            assert aborted is True
    
            nonlocal aborted_called
            aborted_called = True
    
        await driver.submit(0, "sh", "-c", "sleep 3; exit 2")
>       await poll(driver, {0}, started=started, finished=finished)

/mnt/scratch/f_scout_ci/actions-runner-03/_temp/test_root/tests/integration_tests/scheduler/test_generic_driver.py:105: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/mnt/scratch/f_scout_ci/actions-runner-03/_temp/test_root/tests/utils.py:135: in poll
    await finished(event.iens, event.returncode, event.aborted)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

iens = 0, returncode = 2, aborted = False

    async def finished(iens, returncode, aborted):
        assert iens == 0
>       assert returncode in expected_returncodes
E       assert 2 in [143, 271]
```
(https://github.com/equinor/komodo-releases/actions/runs/8368245844/job/22912653634)


**To reproduce**
Steps to attempt to reproduce the behaviour:
`pytest tests/integration_tests/scheduler/test_generic_driver.py::test_kill -k OpenPBSDriver --count 800 -n 12`

**Expected behaviour**
Test passes consistently

--- a/tests/integration_tests/scheduler/test_generic_driver.py
+++ b/tests/integration_tests/scheduler/test_generic_driver.py
@@ -99,6 +99,6 @@ async def test_kill(driver, tmp_path):
         nonlocal aborted_called
         aborted_called = True
 
-    await driver.submit(0, "sh", "-c", "sleep 3; exit 2")
+    await driver.submit(0, "sh", "-c", "sleep 60; exit 2")
     await poll(driver, {0}, started=started, finished=finished)
     assert aborted_called

